assets/src/scss/blocks.scss

```diff
@@ -1,6 +1,6 @@
 // Beta blocks
-@import "blocks/ActionsList";
-@import "blocks/PostsList";
+@import "blocks/ActionsList/ActionsList";
+@import "blocks/PostsList/PostsList";
```

Comment:
For consistency, these files should be named `ActionsListStyle` and `PostsListStyle`

Reply:
Updated!